src/components/Pill.svelte

```diff
@@ -0,0 +1,12 @@
+<script>
+  export let data;
+  export let pillColors;
```

Comment:
It's clearer to have each parameter (bgColor and borderColor) passed in individually, rather than as an object.

Also, they should have reasonable default values (e.g. a neutral grey color)